sdk/communication/Azure.Communication.Common/src/CommunicationTokenRefreshOptions.cs

```diff
@@ -15,31 +15,27 @@ public class CommunicationTokenRefreshOptions
     {
         internal bool RefreshProactively { get; }
         internal Func<CancellationToken, string> TokenRefresher { get; }
-        internal Func<CancellationToken, ValueTask<string>> AsyncTokenRefresher { get; }
-
 #nullable enable
-        internal string? InitialToken { get; }
+        /// <summary>The initial token.</summary>
+        public string? InitialToken { get; set; }
```

Comment:
we should remove the `#nullable enable` now that this property is public. We are not supposed to use nullable ref-types in the public interface

Reply:
Thanks, good to know. I've removed #nullable